### DISCUSSION TITLE
Search not 100% working

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1493] [General]
The search function is not finding "South Australia", it is only returning results for "South".  Whether it is the location, street address, suburb, or name of the meeting, or state (New South Wales), but it's not actually searching for South Australia. 

Do you have any advice why it might be excluding "South Australia" from the results. This is the live link to the meetings page:
https://al-anon.org.au/meetings/

Cheers

**Comment by joshreisner** [comment]
when you enter text in the keyword search you're going to match meetings that contain that text.

did you want to match all the meetings in the `SA` region? https://al-anon.org.au/meetings/?region=sa i'd try renaming that to South Australia so they all contain those keywords

then, if you put quotes around "South Australia" in the search, you should only match those meetings that have those words in sequence
